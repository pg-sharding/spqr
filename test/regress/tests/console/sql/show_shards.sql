DROP SHARD sh1;
DROP SHARD sh2;
DROP SHARD sh3;
DROP SHARD sh4;

CREATE SHARD sh1 WITH HOSTS spqr_shard_1:6432,spqr_shard_1_replica:6432;
CREATE SHARD sh2 WITH HOSTS spqr_shard_1:6432,spqr_shard_1_replica:6432;
CREATE SHARD sh3 WITH HOSTS spqr_shard_1:6432,spqr_shard_1_replica:6432;
CREATE SHARD sh4 WITH HOSTS spqr_shard_1:6432,spqr_shard_1_replica:6432;

CREATE SHARD sh1 WITH HOSTS localhost:6432;

SHOW shards;
SHOW hosts;

CREATE SHARD sh5 WITH HOSTS e:6432;

SHOW shards;
SHOW hosts;

DROP SHARD sh5;

SHOW shards;
SHOW hosts;