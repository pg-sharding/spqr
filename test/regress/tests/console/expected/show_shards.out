
		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

SHOW shards;
  listing data shards  
-----------------------
 datashard with ID sh1
 datashard with ID sh2
(2 rows)

